packages/core/useWebWorkerFn/index.ts

```diff
@@ -21,17 +26,12 @@ export interface WebWorkerOptions extends ConfigurableWindow {
  */
 export const useWebWorkerFn = <T extends (...fnArgs: any[]) => any>(
   fn: T,
-  {
-    dependencies = [],
-    timeout,
-    window = defaultWindow,
-  }: WebWorkerOptions = {},
+  { dependencies = [], timeout, window = defaultWindow }: Partial<WebWorkerOptions> = {},
```

Comment:
This is the only main change. Making optinons a Partial type.
It shouldn't be a breaking change